lib/ome/common/base64.h

```diff
@@ -0,0 +1,322 @@
+/*
+ * #%L
+ * OME-COMMON C++ library for C++ compatibility/portability
+ * %%
+ * Copyright © 2006 - 2015 Open Microscopy Environment:
```

Comment:
Oops, that's what you get from copying another header.  I managed to remember to fix it up for the unit test, now fixed here.